### ISSUE TITLE
Support for uncompleted Reverse Dragonriding Races

### ISSUE THREAD

**Issue by Arxareon** [zarillion/handynotes-plugins#288]
**ADDON VERSION**: v31

It has been asked for on the CurseForge page already by others but currently, Dragonriding Race icons are hidden if the normal and Advanced courses are completed and ALL completed stuffs need to be shown in order to still have them shown, even if Reverse courses aren't complete.

There is an argument to be made of having a separate option just for displaying completed Races (even after Reverse courses are done).

**Comment by Ioney** [comment]
i cant replicate this error. are you sure you are on v31? are the reverse races showing on the tooltip if you activate the option to show already completed nodes?

**Comment by Arxareon** [comment]
Right, I'm not quite sure what was wrong as I reinstalled and it now works properly. Reverse courses are listed and the ones I haven't done are shown as incomplete. I did have the latest version because I remember checking but I actually had to reinstall my whole PC since then so.. not sure, maybe CurseForge was bugged or I was blind.
Anyhow, reinstalling the addon should probably fix it if anyone else has a similar issue.

Thanks for looking into this, anyways! :)
